### PR TITLE
Sidenav description

### LINKED ISSUE #1185: Batch edit sidenav item descriptions
In batch edit container, each side nav item has a description about which fields are missing

**AC**
- a function somewhere exists that takes in an enrollment and returns a missing info description string
- If enrollment is missing birth certificate or income determination (or re-determination), I see those missing fields named, and then I see a count of how many other fields are missing.
- If enrollment is not missing birth certificate or income determination, but are missing other fields, I only see the count of how many fields are missing: "5 fields missing".

### PR DESCRIPTION
**Pull request by mmazanec22**
Closes #1185

### CODE REVIEW

File: src/Hedwig/ClientApp/src/containers/BatchEdit/BatchEdit.tsx [hunk 1]
```diff
@@ -24,6 +24,8 @@ const BatchEdit: React.FC<BatchEditProps> = ({ match: { params: sectionId } }) =
 		orgId: getIdForUser(user, 'org'),
 		startDate: startDate.toDate(),
 		endDate: endDate.toDate(),
+		// Need to include these fields so we can indicate how many are missing data
+		include: ['child', 'family', 'determinations', 'fundings'],
```
Comment (jlhogan) [inline src/Hedwig/ClientApp/src/containers/BatchEdit/BatchEdit.tsx:28]: `include` QS param has been deprecated in the API  (moving away from pretending this is GraphQL), so this should not be necessary -- if an of this data is missing in the response, that will need to be a backend change

File: src/Hedwig/ClientApp/src/containers/BatchEdit/BatchEdit.tsx [hunk 2]
```diff
@@ -34,10 +36,11 @@ const BatchEdit: React.FC<BatchEditProps> = ({ match: { params: sectionId } }) =
 		return <>Loading...</>;
 	}
 
-	const needInfoEnrollments = (enrollments || []).filter(
-		(e) =>
+	const needInfoEnrollments = (enrollments || []).filter((e) => {
+		return (
```
Comment (jlhogan) [inline src/Hedwig/ClientApp/src/containers/BatchEdit/BatchEdit.tsx:40]: why this change?

File: src/Hedwig/ClientApp/src/containers/BatchEdit/SingleEnrollmentEdit.tsx [hunk 3]
```diff
@@ -145,6 +145,7 @@ export const SingleEnrollmentEdit: React.FC<SingleEnrollmentEditProps> = ({
 					</div>
 				</div>
 				<div className="margin-top-1">
+					{/* TODO: why isn't this just a link? */}
```
Comment (jlhogan) [inline src/Hedwig/ClientApp/src/containers/BatchEdit/SingleEnrollmentEdit.tsx:148]: I dont know. Nothing i can think of / remember

File: src/Hedwig/ClientApp/src/utils/validations/getMissingInfoPrettyString.ts [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+		missingInfoString = splitCamelCase(birthCertValidationErrors[0].field);
+	}
+
+	if (
+		incomeDeterminationValidationErrors.length > 1 ||
+		!enrollment.child ||
+		!enrollment.child.family
+	) {
+		// Income household size, income and date -> Income determination)
+		missingInfoString += ', income determination';
```
Comment (jlhogan) [inline src/Hedwig/ClientApp/src/utils/validations/getMissingInfoPrettyString.ts:71]: what if only income determination, not birth certificate is missing? will the string be like ", income determination fields" ?

File: src/Hedwig/ClientApp/src/containers/BatchEdit/BatchEdit.tsx (hunk 2 above)
Comment (mmazanec22) [reply src/Hedwig/ClientApp/src/containers/BatchEdit/BatchEdit.tsx:40]: Was probably console logging something-- will change

File: src/Hedwig/ClientApp/src/utils/validations/getMissingInfoPrettyString.ts [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+	}
+
+	if (remainingValidationErrors.length) {
+		if (
+			birthCertValidationErrors.length === 0 &&
+			incomeDeterminationValidationErrors.length === 0
+		) {
+			// If enrollment is not missing birth certificate or income determination, but are missing other fields,
+			// I only see the count of how many fields are missing: "5 fields missing".
+			return `${remainingValidationErrors.length} fields`;
```
Comment (cailyncodes) [inline src/Hedwig/ClientApp/src/utils/validations/getMissingInfoPrettyString.ts:84]: Does "missing" need to be added to this return string?

Comment (mmazanec22) [reply src/Hedwig/ClientApp/src/utils/validations/getMissingInfoPrettyString.ts:84]: No and I should probably take out these comments-- the AC didn't exactly match the prototype, so I followed the prototype
